edward/util/random_variables.py

```diff
@@ -714,6 +715,93 @@ def get_variables(x, collection=None):
   return list(output)
 
 
+def is_independent(a, b, condition=None):
+  """Assess whether a is independent of b given the random variables in
+  condition.
+
+  Implemented using the Bayes-Ball algorithm[1].
+
+  Args
+    a : RandomVariable or list of RandomVariable
```

Comment:
Args are written as, e.g., `a: RandomVariable or list of RandomVariable.` Note the removed space and added period. When in doubt, you can also double check the generated API docstrings by checking out the README in `docs/`, or consult with the other function docstrings.